tests/libvmi/vmi.go

```diff
@@ -145,3 +145,19 @@ func WithResourceCPU(value string) Option {
 		vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceCPU] = resource.MustParse(value)
 	}
 }
+
+// WithSerialBios if set the BIOS output will be transmitted over serial.
+func WithSerialBios(secureBoot bool) Option {
```

Comment:
The `secureBoot` option is a leftover and should be removed entirely imo

Reply:
Done.